prismlauncher

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# discord RPC
+for i in {0..9}; do
+    test -S "$XDG_RUNTIME_DIR"/discord-ipc-"$i" || ln -sf {app/com.discordapp.Discord,"$XDG_RUNTIME_DIR"}/discord-ipc-"$i";
+done
+
+export PATH="${PATH}${PATH:+:}/app/utils/gamescope/bin:/usr/lib/extensions/vulkan/MangoHud/bin"
+export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}${LD_LIBRARY_PATH:+:}/usr/lib/extensions/vulkan/MangoHud/\$LIB/"
```

Comment:
You don't need to add MangoHud libs to `LD_LIBRARY_PATH`, it's built with `RPATH`s set exactly to avoid the need to alter `LD_LIBRARY_PATH`.
If MangoHud doesn't work without it - it's a bug, please report it [here](https://github.com/flathub/org.freedesktop.Platform.VulkanLayer.MangoHud/issues).

Reply:
The application has built-in support for MangoHud. For this to work, it needs to be able to find libMangoHud.so or libMangoHud_dlsym.so to enable the feature.
See https://github.com/PolyMC/PolyMC/blob/ebbc1f67e02e62b8024280232cfb936468afd395/launcher/Application.cpp#L1548-L1558

Reply:
ACK. Although I'd like a point out that `dlopen`'ing `libMangoHud.so` isn't a proper way to detect if MangoHud is installed, because `libMangoHud.so` isn't supposed to be present in the linker search path at all. For example, Fedora explicitly prohibits installing unversioned libs to default libdir, so this check won't work with Fedora's `mangohud` package.

Reply:
Okay. I might look into fixing up that detection mechanism. I think it is fine for now.

Reply:
Created an issue on upstream: https://github.com/PrismLauncher/PrismLauncher/issues/165

Reply:
Yeah, it's not a blocker for this flatpak, just something to keep in mind.